doc/profile_changes_SLE15.md

```diff
@@ -0,0 +1,150 @@
+- FW_DEV_{DMZ, EXT, INT}
+- FW_SERVICES_{DMZ, INT, EXT}_{TCP, UDP, IP}
+- FW_LOG_ACCEPT_CRIT
+- FW_LOG_DROP_CRIT
+- FW_LOG_DROP_ALL
+- FW_MASQUERADE
+
+The following examples will show with defails the conversion of each property.
+
+#### Whether firewalld should be enabled and running after the installation
+
+```xml
+  <firewall>
+    # These attributes are the only ones that are completely compatible in both schemas
+    <enable_firewall>true</enable_firewall>
+    <start_firewall>true</start_firewall>
+  </>
+```
+
+Both firewalls are zone-based with a different predefined set of rules and level
+of trust for network connections.
+
+Whereas SuSEfIrewall2 has 3 zones by default (DMZ, EXT, INT) firewalld provides
+a few more (block, dmz, drop, external, home, internal, public, trusted, work).
+In SuSEFirewall2 the default zone is the external one (EXT) but also allows the
+use of the special keyword `any` to assign all the interfaces that are not listed
+anywhere to a specified zone.
+
+#### Assignation of interfaces to zones
+
+The DMZ zone will be mapped to the 'dmz' zone, the EXT zone will be mapped to
+the 'public' zone if FW_MASQUERADE is disabled or to the 'external' if it is
+enabled and finally the INT zone will be mapped to the 'internal' zone if
+FW_PROTECT_FROM_INT is true or to the 'trusted' zone if not.
+
+
+**Default assignation**
+```xml
+<firewall>
+  FW_DEV_DMZ="any eth0"
+  FW_DEV_EXT="eth1 wlan0"
+  FW_DEV_INT="wlan1"
+</firewall>
+```
+
+```xml
+<firewall>
+  <default_zone>dmz</default_zone>
+  <zones>
+    <zone>
+      <name>dmz</name>
+      <interfaces>eth0</interfaces>
+    </zone>
+    <zone>
+      <name>public</name>
+      <interfaces>eth1</interfaces>
+    </zone>
+    <zone>
+      <name>trusted</name>
+      <interfaces>wlan1</interfaces>
+    </zone>
+  </zones>
+
+```
+
+**With masquerading and protecting internal zone**
+
+```xml
+<firewall>
+  FW_DEV_DMZ="any eth0"
+  FW_DEV_EXT="eth1 wlan0"
+  FW_DEV_INT="wlan1"
+  FW_MASQUERADE="yes"
+  FW_PROTECT_FROM_INT="yes"
+</firewall>
+```
+
+```xml
+<firewall>
+  <default_zone>dmz</default_zone>
+  <zones>
+    <zone>
+      <name>dmz</name>
+      <interfaces>eth0</interfaces>
+    </zone>
+    <zone>
+      <name>external</name>
+      <interfaces>eth1</interfaces>
+    </zone>
+    <zone>
+      <name>internal</name>
+      <interfaces>wlan1</interfaces>
+    </zone>
+  </zones>
+
+```
+
+#### NTP Configuration
+
+- https://susedoc.github.io/doc-sle/develop/SLES-autoyast/html/configuration.html#Configuration.Network.Ntp
+
+The time server synchronization daemon ntpd has been replaced with the more
+modern daemon Chrony.
+
+This change means that AutoYaST files with an ntp_client section need to be
+updated to a new format for this section.
+
+Instead of containing low level configuration options, is now composed of a set
+of high level ones that are applied on top of the default settings.
+
+And here is how the new (and nicer) configuration looks like:
```

Comment:
I guess we also have to mention that we do not support NTP authentication options (keyfile /etc/chrony.keys), and maybe something else.

Reply:
And also that chrony is not availabe in unregistered SLES (or one without all-packages DVD), we need separate section for that anyway, so maybe adding reference to it afterwards.